### PR TITLE
Isolate handControllerGrab.js changes from bow branch

### PR DESCRIPTION
**Pull request by imgntn**
This PR adds a listener for messages to disable a hand when using a two-handed object.  It also makes the beams entities ungrababble and non-colliding.

### CODE REVIEW

File: examples/controllers/handControllerGrab.js [hunk 1]
```diff
@@ -1218,12 +1213,35 @@ mapping.from([Controller.Standard.LB]).peek().to(leftController.bumperPress);
...
 }
 
+Messages.subscribe('Hifi-Beam-Disabler');
+
+handleBeamDisablerMessages = function(channel, message, sender) {
+
+    if (sender === MyAvatar.sessionUUID) {
+        handToDisable = message;
+        if (message === 'left') {
+            handToDisable = 1;
```
Comment (sethalves) [inline examples/controllers/handControllerGrab.js]: I find it weird that the 1 and 0 constants here suggest the opposite of what I would expect, given the constants at the top:
var RIGHT_HAND = 1;
var LEFT_HAND = 0;

**Comment by imgntn** [comment]
@sethalves i updated the bow api to do the hand switching so disabling should be clearer here now as well

File: examples/controllers/handControllerGrab.js [hunk 2]
```diff
@@ -1218,12 +1212,35 @@ mapping.from([Controller.Standard.LB]).peek().to(leftController.bumperPress);
...
-    leftController.update();
+    if (handToDisable !== LEFT_HAND) {
+        leftController.update();
+    }
+    if (handToDisable !== RIGHT_HAND) {
+        rightController.update();
+    }
 }
 
+Messages.subscribe('Hifi-Hand-Disabler');
```
Comment (sethalves) [inline examples/controllers/handControllerGrab.js]: I'm not sure how this works, but if you grab a bow with your left hand, is this going to disable the right hand of everyone in the zone?

File: examples/controllers/handControllerGrab.js [hunk 3]
```diff
@@ -1218,12 +1212,35 @@ mapping.from([Controller.Standard.LB]).peek().to(leftController.bumperPress);
...
+
+handleHandDisablerMessages = function(channel, message, sender) {
+
+    if (sender === MyAvatar.sessionUUID) {
```
Comment (sethalves) [inline examples/controllers/handControllerGrab.js]: ah, only if the sender is me.

**Pull request merged**